lighthouse-core/audits/theme-color-meta.js

```diff
@@ -38,7 +38,7 @@ class ThemeColor extends Audit {
    */
   static audit(artifacts) {
     const themeColorMeta = artifacts.ThemeColor;
-    if (!themeColorMeta) {
+    if (themeColorMeta === null || themeColorMeta === -1) {
```

Comment:
no, it's just that `themeColorMeta` uses null for a signal (since [`Element.getAttribute()` returns null](https://github.com/GoogleChrome/lighthouse/blob/93cbff3d88679146d0e7daf62bbde0c926deab64/lighthouse-core/lib/element.js#L33) on not found) and I was going to wait on updating that until we standardize on gatherer error handling. So, mostly just making it explicit here that's it's not a check against `undefined` :)